prem_utils/models.json

```diff
@@ -593,6 +593,57 @@
                     "coming_soon": true
                 }
             ]
+        },
+        {
+            "provider": "anyscale",
+            "models": [
+                {
+                    "slug": "meta-llama/Llama-2-7b-chat-hf",
```

Comment:
Same here. Double check. Slug should be unique.

Reply:
Yes I added the `anyscale/` prefix